### PR TITLE
Add nodeAlias property to topNNode model

### PR DESCRIPTION
**Pull request by faizanjanjua**
## Description
LOK-2239: Add nodeAlias property to topNNode model
DONE
 will add node alias as part of model, when available show node alias on Top Node table on the Home screen ( when not available use label)

## Jira link(s)
- https://opennms.atlassian.net/browse/LOK-2304

## Checklist
* [ ] Follows Lōkahi's [development guidelines.](https://github.com/OpenNMS-Cloud/lokahi/wiki/Development-Guidelines)
* [ ] Appropriate reviewer(s) have been selected.
* [ ] Jira issue(s) have been updated to "In Review".
* [ ] Includes [appropriate tests.](https://github.com/OpenNMS-Cloud/lokahi/wiki/Test-Strategy)
* [ ] Documentation has been updated as necessary.
* [ ] Notify devops of changes to the Charts.
* [ ] Notify documentation team of any changes to names of screens or features (affects URLs).

### CODE REVIEW

File: ui/src/types/graphql.ts [hunk 1]
```diff
@@ -1623,7 +1623,10 @@ export const ListMinionMetricsDocument = { 'kind': 'Document', 'definitions': [{
...
+
```
Comment (Shahbaz-dataq) [inline ui/src/types/graphql.ts:1626]: Remove these extra spaces

File: ui/src/types/graphql.ts [hunk 2]
```diff
@@ -1623,7 +1623,10 @@ export const ListMinionMetricsDocument = { 'kind': 'Document', 'definitions': [{
...
+
```
Comment (Shahbaz-dataq) [inline ui/src/types/graphql.ts:1629]: Same remove extra spaces

File: ui/src/components/Dashboard/DashboardTopNodesTable.vue [hunk 3]
```diff
@@ -53,6 +53,7 @@
               :key="topNode.nodeLabel + topNode.avgResponseTime"
             >
               <td>{{ topNode.nodeLabel }}</td>
+              <td>{{ topNode.nodeAlias == "" ? topNode.nodeLabel  : topNode.nodeAlias  }}</td>
```
Comment (synqotik) [inline ui/src/components/Dashboard/DashboardTopNodesTable.vue:56]: Can just be:

```
topNode.nodeAlias || topNode.nodeLabel
```

**Review by synqotik — CHANGES_REQUESTED** [review]

**Review by synqotik — APPROVED** [review]
LGTM

**Pull request merged**